robottelo/helpers.py

```diff
@@ -253,3 +253,20 @@ def remove_katello_ca(hostname=None):
             )
             if result.return_code != 0:
                 raise AssertionError('Failed to reset the rhsm.conf')
+
+
+def add_remote_execution_ssh_key(hostname, key_path=None, **kwargs):
```

Comment:
So basically, we should use Jenkins key to connect to both servers. There is a chance that we use Beaker password to connect to Sat machine and Jenkins key to VM, but I don't think that is supported at that moment